tests/v2/validation/schemas/schemas_test.go

```diff
@@ -0,0 +1,322 @@
+func (sc *SchemaChangesTestSuite) TestPreferredVersionCheckLocalCluster() {
+	subSession := sc.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Access the schemas endpoint on the local cluster and verify that it can be accessed successfully without any errors.")
+	schemasCollection, err := sc.client.Steve.SteveType(schemaSteveType).List(nil)
+	require.NoError(sc.T(), err)
+	log.Infof("Number of schemas: %d", len(schemasCollection.Data))
+
+	log.Info("Verify that the version in each schema is the preferred version.")
+	failedSchemaPreferredVersionCheck, err := checkPreferredVersion(sc.client, schemasCollection, localCluster)
+	require.NoError(sc.T(), err)
+	require.Emptyf(sc.T(), failedSchemaPreferredVersionCheck, "Failed preferred version checks: %v", failedSchemaPreferredVersionCheck)
+}
+
+func (sc *SchemaChangesTestSuite) TestPreferredVersionCheckDownstreamCluster() {
+	subSession := sc.session.NewSession()
+	defer subSession.Cleanup()
+
+	steveAdminClient, err := sc.client.Steve.ProxyDownstream(sc.cluster.ID)
+	require.NoError(sc.T(), err)
+
+	log.Info("Access the schemas endpoint on the downstream cluster and verify that it can be accessed successfully without any errors.")
+	schemasCollection, err := steveAdminClient.SteveType(schemaSteveType).List(nil)
+	require.NoError(sc.T(), err)
+	log.Infof("Number of schemas: %d", len(schemasCollection.Data))
+
+	log.Info("Verify that the version in each schema is the preferred version.")
+	failedSchemaPreferredVersionCheck, err := checkPreferredVersion(sc.client, schemasCollection, sc.cluster.ID)
+	require.NoError(sc.T(), err)
+	require.Emptyf(sc.T(), failedSchemaPreferredVersionCheck, "Failed preferred version checks: %v", failedSchemaPreferredVersionCheck)
+}
+
+func (sc *SchemaChangesTestSuite) TestCRDCreateDeleteOperation() {
+	subSession := sc.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a new Custom Resource Definition (CRD).")
+	err := createCRD(sc.client, crdCreateFilePath)
+	require.NoError(sc.T(), err)
+
+	log.Info("Access the schemas endpoint for the newly created CRD and verify that it can be accessed successfully without any errors.")
+	err = kwait.PollUntilContextTimeout(context.Background(), twoSecondTimeout, defaults.FiveSecondTimeout, false, func(ctx context.Context) (done bool, pollErr error) {
+		_, pollErr = getSchemaByID(sc.client, localCluster, customSchemaID)
+		if pollErr != nil {
+			return false, pollErr
+		}
+		return true, nil
+	})
+	schemaResponse, err := getSchemaByID(sc.client, localCluster, customSchemaID)
+	require.NoError(sc.T(), err)
+
+	log.Info("Verify that the schema's resourceFields field is set to null.")
+	id := schemaResponse["id"].(string)
+	require.Nil(sc.T(), schemaResponse["resourceFields"], "ResourceFields should be null for schema "+id)
+
+	log.Info("Verify that the schemas endpoint for the new CRD does not list the child schemas and trying to access any of the child schemas results in a '404 Not Found' error.")
```

Comment:
can we condense this to just be `Verify that the schemas endpoint for the new CRD does not list the child schemas`